extensions/2.0/Khronos/KHR_materials_sss/README.md

```diff
@@ -0,0 +1,195 @@
+                "scatterColor": [ 0.572, 0.227, 0.075 ]
+            }
+        }
+    }
+]
+```
+
+## Properties
+
+The extension defines the following parameters to describe the scattering behavior.
+
+| | Type | Description | Required |
+|-|------|-------------|----------|
+| **scatterDistance** | `number` | Average distance in meters that light travels in the medium before it collides with a particle and scatters | No, default: +Infinity |
+| **scatterColor** | `number[3]` | Color as a result of scatter events | No, default: `[0, 0, 0]` |
+
+
+## Scattering
+
+In rendering literature, the scattering behavior is most commonly described by the scattering coefficient σ<sub>s</sub>. The scattering coefficient is a probability density with the unit 1/m<sup>2</sup> and values in the range [0, inf]. The sum of absorption and scattering coefficients is referred to as the attenuation (or extinction) coefficient
+
+σ<sub>t</sub> = σ<sub>a</sub> + σ<sub>s</sub>
+
+Analogous to the parameterization of the absorption coefficient in `KHR_materials_volume`, this extension parameterizes the scattering coefficient in terms of *scattering color c<sub>s</sub>* and *scattering distance d<sub>s</sub>*. 
+
+σ<sub>s</sub> = -log(c<sub>s</sub>) / d<sub>s</sub>
+
+The definition of the scattering coefficient σ<sub>s</sub> replaces the zero constant in `KHR_materials_volume`. As the volume extension already uses the notation of *attenuation coefficient σ<sub>t</sub>*, the derivation of the transmittance function is unchanged
+
+T(x) = e<sup>-σ<sub>t</sub>x</sup>
+
+Instead of only taking into account absorption, T now corresponds to a change in radiance along a path as light travels through a medium with absorbing **and** scattering particles.
+
+
+<figure style="text-align:center">
+<img src="./figures/diffuse-sss.png"/>
+<figcaption><em>A simple, diffuse-only material (left) and a material that makes use of subsurface scattering (right). The base color of the diffuse material is set to the same color as the subsurface color of the subsurface scattering material. Due to the albedo mapping the final color of the object is very similar.</em></figcaption>
+</figure>
+
+## Phase Function
+
+The phase function p used for scattering inside the medium is isotropic. For any pair of incident and outgoing directions k<sub>1</sub> and k<sub>2</sub>, p(k<sub>1</sub>, k<sub>2</sub>) = 1 / (4π).
+
+## Scattering, Translucency & Transmission
+
+For best results, we recommend using `KHR_materials_translucency` instead of `KHR_materials_transmission` in case the medium exhibits strong subsurface scattering (large values for the scattering coefficient σ<sub>s</sub>). Examples for these dense materials are skin or candle wax. The visual difference between translucency and transmission is small in this case, as the path a light travels is dominated by volume scattering. The scattering interaction at the volume boundary has only a small effect on the final result. 
```

Comment:
```suggestion
For best results, we recommend using `KHR_materials_diffuse_transmission` instead of `KHR_materials_transmission` in case the medium exhibits strong subsurface scattering (large values for the scattering coefficient σ<sub>s</sub>). Examples for these dense materials are skin or candle wax. The visual difference between translucency and transmission is small in this case, as the path a light travels is dominated by volume scattering. The scattering interaction at the volume boundary has only a small effect on the final result. 
```